### PR TITLE
Change the way of displaying boolean metadata values.

### PR DESCRIPTION
**Pull request by ewelinajozwik**
Implements [FAIRSPC-48](https://thehyve.atlassian.net/browse/FAIRSPC-48)

[FAIRSPC-48]: https://thehyve.atlassian.net/browse/FAIRSPC-48?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: projects/mercury/src/common/utils/genericUtils.js [hunk 1]
```diff
@@ -163,6 +165,16 @@ export const formatDateTime = value => {
     return isToday ? timeFormatter.format(date) : dateFormatter.format(date);
 };
 
+export const renderBooleanIcon = value => {
+    if (value === true || value === 't') {
```
Comment (tgreenwood) [inline projects/mercury/src/common/utils/genericUtils.js]: probably it would make sense to turn it to lower case first and then compare

Comment (ewelinajozwik) [reply projects/mercury/src/common/utils/genericUtils.js]: I changed this part

File: projects/mercury/src/metadata/common/values/ReferringValue.js [hunk 2]
```diff
@@ -18,8 +17,10 @@ export const ReferringValue = ({property, entry}) => {
         switch (property.datatype) {
             case DATETIME_URI:
                 return formatDateTime(value.value);
-            case BOOLEAN_URI:
-                return <Switch checked={!!value.value} readOnly />;
+            case BOOLEAN_URI: {
```
Comment (tgreenwood) [inline projects/mercury/src/metadata/common/values/ReferringValue.js:20]: maybe there were two places, I would still make it case-insensitive

Comment (ewelinajozwik) [reply projects/mercury/src/metadata/common/values/ReferringValue.js:20]: you're right, I changed it now

**Review by tgreenwood — APPROVED** [review]

**Pull request merged**